subcommands/devices/list.go

```diff
@@ -142,6 +144,8 @@ func init() {
 	}
 	cmd.AddCommand(listCmd)
 	listCmd.Flags().BoolVarP(&deviceMine, "just-mine", "", false, "Only include devices owned by you")
+	listCmd.Flags().BoolVarP(&deviceOnlyProd, "only-prod", "", false, "Only include production devices")
+	listCmd.Flags().BoolVarP(&deviceOnlyNonProd, "only-non-prod", "", false, "Only include non-production devices")
```

Comment:
Is the `prod=0` going to output only non prod devices or all devices?

Reply:
Yes, the `--only-prod=0` is going to print all devices, which is kind of expected.

Although, I only expect the user to do `--only-prod` or simply omit that flag.

Reply:
Then the `only-non-prod` is not quite correct name.